lib/src/api/engine/remote/ws/wasm.rs

```diff
@@ -217,7 +218,8 @@ pub(crate) fn router(
 							}
 							let payload = Value::from(request);
 							trace!("Request {payload}");
-							Message::Binary(payload.into())
+							let payload = serialize(&payload).unwrap();
```

Comment:
I know it's very unlikely that we will run into a serialisation error here but now that we can handle it, I would like to do so. Any chance you can leave a `TODO` comment here so I can handle this later?

Reply:
Hey @rushmorem yes we definitely want to use revisioned serialization across the API boundary. We just need to work out how to support some of the `Error` types. Adding `revisioned` to the `Error` type works, except for some of the `Bincode`, `HTTP`, `TiKV`, and so on errors...